### PR TITLE
Please review Monsters

### PR DESCRIPTION
**Pull request by akshaykgarg**

### CODE REVIEW

**Review by NamidaSai — CHANGES_REQUESTED** [review]
Comments for "designer-side" improvements even though the current code makes sense on programmer side. Can wait for next push. 

Only thing required before merging: the rest of the codebase is passing in a char to the Monster.cpp constructor not a string.

**Comment by NamidaSai** [comment]
One more thought: we will actually be using the full name type to display event messages in the player log. "You attacked a Troll!" etc. So yeah, need the full name back from programmer side as well not only for being nice to theoretical designers.

File: output/Data/monsters.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+    "T":{
```
[the remark this answers is not in the document]
Comment (akshaykgarg) [reply output/Data/monsters.json:2]: The json format has been this way now

**Review by NamidaSai — APPROVED** [review]
Good to merge

**Pull request merged**